src/app/auth/auth.component.ts

```diff
@@ -134,6 +134,10 @@ export class AuthComponent implements OnInit, OnDestroy {
     return this.authState === AuthState.ConfirmOAuthUser;
   }
 
+  isRepoSet(): boolean {
+    return !(this.phaseService.currentRepo === undefined);
+  }
```

Comment:
I've renamed the methods as suggested.

I also moved `isRepoSet()` in `AuthService` into `PhaseService`, however, I've left the `isRepoSet()` in `AuthComponent` as is as I'm reluctant to make `phaseService` in `AuthComponent` public so that its accessible in `auth.component.html`. I've changed  it to make use of `isRepoSet()` in `PhaseService` though.

As for `SetRepo` and `handleSetRepoSuccess`, I've left them in `AuthService` for now as aside from calling `PhaseService::initializeCurrentRepository()`, they mostly concern  cleaning up before routing to `Phase.issuesViewer`. I'm not too sure how I should break it down further between `AuthService` and `PhaseService`.